### PR TITLE
PhpUnitTestCaseStaticMethodCallsFixer - fix for having property with name as method to update

### PR DESCRIPTION
**Pull request by kubawerlos**
Fixes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4312

Pinging @Slamdunk (as original author) and @ro0NL (as reporter) for review.

### CODE REVIEW

**Comment by ro0NL** [comment]
seems to work for me :+1:

File: tests/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixerTest.php [hunk 1]
```diff
@@ -433,6 +433,19 @@ public function baz()
...
                 ],
             ],
+            [
+                <<<'EOF'
+<?php
+class HavingPropertyWithNameAsMethodToUpdateTest extends PHPUnit
+{
+    public function foo()
+    {
+        $this->attribute = 42;
```
Comment (Slamdunk) [inline tests/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixerTest.php]: Test and fix is ok, but I think it isn't 100% clear enough what you are testing, for someone who look at this the first time. The class name of course helps, but I would use a more wide spread `$this->assertSame = 42` for example.

Comment (SpacePossum) [reply tests/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixerTest.php]: :+1:  we can have both I think :)

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**